assets/scss/_variables_project.scss

```diff
@@ -8,3 +8,39 @@ $td-fonts-serif: BlinkMacSystemFont, -apple-system, "Segoe UI", "Roboto", "Oxyge
 $td-enable-google-fonts: false;
 $light: #929292 !default;
 $dark: #333 !default;
+$mobile-width: 767px;
+
+/*
+    Styling for privacy notice watermark
+    in privacy-notice.html
+*/
+#privacy-notice {
+    background-color: rgba(84, 73, 75, 0.9); 
+    position: fixed; 
+    top: 100%; 
+    right: 0%; 
+    width: 500px; 
+    margin: -15rem 0 0 0; 
+    height: 15rem;
```

Comment:
`min-height: 10rem;` avoids an odd gap at the bottom but allows it to increase if needed